web/client/plugins/featuregrid/FeatureEditor.jsx

```diff
@@ -24,6 +24,7 @@ import {modeSelector, changesSelector, newFeaturesSelector, hasChangesSelector,
 
 import {getPanels, getHeader, getFooter, getDialogs, getEmptyRowsView, getFilterRenderers} from './panels/index';
```

Comment:
Please revert this change the Fallback component must be isolated and as much simple as possible and not be included inside the main component

Reply:
Done